aion/test/models/question_test.exs

```diff
@@ -15,4 +15,17 @@ defmodule Aion.QuestionTest do
     changeset = Question.changeset(%Question{}, @invalid_attrs)
     refute changeset.valid?
   end
+
+  def get_question(question_id) do
+    Repo.get(Question, question_id)
+  end
+
+  def get_random_question(category_id) do
+    query = from q in Question, where: q.subject_id == ^category_id
+    question =
+      query
+      |> Repo.all()
+      |> Enum.random()
```

Comment:
Here you omit the parentheses: https://github.com/pmrukot/aion/pull/67/files#diff-fcb2b290b530eae3a0bbb6b044f1cfbeR32

I think we need to settle down on way of piping through one-arity functions. Elixir style guide recommends doing so but Im not fully convinced:
https://github.com/christopheradams/elixir_style_guide#parentheses-pipe-operator